docker-compose.yml

```diff
@@ -5,6 +5,8 @@ services:
     privileged: true
     build: .
     ports: ["5000:5000"]
+    environment:
+      - LOGGING_LEVEL=10
```

Comment:
This enables debug-logging by default... I don't think that's a good idea. `20` would be a more appropriate default.